### ISSUE TITLE
After running "remesh_smooth", the new grid did not save the vertex colors of the old grid. 

### ISSUE THREAD

**Issue by skylinestars** [BrunoLevy/geogram#157]
Excuse me，After running "remesh_smooth", the new grid did not save the vertex colors of the old grid. How should I calculate the vertex colors of the new grid

**Comment by BrunoLevy** [comment]
Yes, the result of `remesh smooth` is a mesh that is completely different from the original mesh, so it does not have the initial attributes. You will have to transfer the attributes, which you can do by creating an AABB (axis-aligned bounding box tree) and find the nearest triangle for each vertex of the remesh, then interpolate the colors from the mesh. Examples of such attribute transfers are in [`geogram/mesh/mesh_baking.cpp`](https://github.com/BrunoLevy/geogram/blob/main/src/lib/geogram/mesh/mesh_baking.cpp)
